README.md

```diff
@@ -1,47 +1,127 @@
-# Project-Pride-Land
+# 🌻 Project-Pride-Land
 
-## Introduction
+## 🐐 Goal
+This is the front-end repository for our project Pride Farm! Our mission is to assist the farm NPO ぷらいどらんど in making a website for its community;
+a place where anyone and everyone can come together and particpate in farming regardless of their health conditions.
 
-## Built with:
+## 💬 Mission
+We want to be able to express what ぷらいどらんど stands for and ensure that their message of companionship, acceptance, and community is displayed. Not only that, but we also strive to make this website easy to maintain and understand for the staff at the NPO.
+
+## 💻 Built with:
 
 Our Front-End Tech stack: <br>
 
 ```
 • React-Vite
 • Typescript
 • TailwindCSS
+• PHASER
 ``` 
+# 👷 Contributing
+Make sure to fork our repo, and fill out the informaiton in the PR message template! If you have any questions feel free to reach out to any of our members.
 
-### Setting up Development
+# 🔧 Setting up Development
 
 Here's a quick guide and run down on how to setup environment and start developing the project.
 
-#### Initial Setup:
+### Prerequisites
+```
+• Node / NPM
+• Plugins for React and Tailwind, such as Tailwind CSS IntelliSense
+```
+
+### Initial Setup:
 
 ```
-$ git clone:  https://github.com/pride-land/frontend-pride-land
-$ cd frontend-pride-land
+# install dependencies
+$ npm install
+
+# start a dev server running with auto-reloading on localhost:5173 
+$ npm run dev
 ```
 
-#### Quick start:
+# 📁 Folder Structure
+
+### public
+* contains assets and pictures used throught various parts of the webapp
+    * **/assets** : contains all the assets utilized in our Phaser game
+### src
+* holds all of our pages, components, the game and webapp itself, etc.
```

Comment:
Now this is awesome. So beginner friendly. In the future you can look into something like gitbook to have documentation like this in another place. But 💯 this is awesome